### ISSUE TITLE
Does this copy images from other hosts?

### ISSUE THREAD

**Issue by stevevance** [svycka/sv-images#8]
An advantage of Imgix is that it copies images that aren't hosted on your own server and serves it from their CDN. 

Can `sv-images` copy images from other hosts and serve it locally?

**Comment by svycka** [comment]
@stevevance yes it can, this module by default uses https://flysystem.thephpleague.com/ so it supports all storage adapters supported by `flysystem` but if you want you can create your own storage adapter.

also I am not using this in production so far and also project needs more tests.
